administrator/templates/atum/scss/_variables.scss

```
@@ -122,7 +122,7 @@ $display2-size:                    .875rem;
 $content-font-size:                0.875rem;
 $label-font-size:                  0.85rem;
 $badge-font-size:                  0.75rem;
-$fa-css-prefix:                    fa;
+$fa-css-prefix:                    fas;
```

```suggestion
$fa-css-prefix:                    fa;
```

This has to be reverted. It is generating this:

```
.fas-eye:before {
  content: "\f06e"; }
```
when it should be:
```
.fa-eye:before {
  content: "\f06e"; }
```

---

Thats one of the ones I was talking about.  It's actually a .far.  It's the exception to the rule so to speak.
https://fontawesome.com/icons/eye?style=regular

---

It is not about `.far` or `.fas` style. This is the icon name prefixed `fa-`.

---

talking about here? https://github.com/joomla/joomla-cms/blob/639761690bbd4c7dc005329c1e14b42a3d3297ba/build/media_source/system/scss/_icomoon.scss#L752

---

In other words, `$fa-css-prefix` is the prefix to append the icon name.

---

OH SNAP! I see what you mean ... ooops.